src/pages/registration/RegistrationForm.tsx

```diff
@@ -142,7 +146,12 @@ export const RegistrationForm = ({ identifier }: RegistrationFormProps) => {
                   </ErrorBoundary>
                 )}
               </Box>
-              <RegistrationFormActions tabNumber={tabNumber} setTabNumber={setTabNumber} validateForm={validateForm} />
+              <RegistrationFormActions
+                tabNumber={tabNumber}
+                setTabNumber={setTabNumber}
+                validateForm={validateForm}
+                hasChangedNviValues={compareRegistrationAndValues(registration, values)}
```

Comment:
Nå vil denne funksjonen kjøres for hver render, men vi ønsker bare å gjøre den sjekken når brukeren prøver å lagre. Ikke noe stort problem nå når det bare er sjekk på år, men når det blir mer kompleksistet og mange sjekker der så kommer nok dette til å føre til at hele formen henger vil jeg tro (siden rerender trigges ganske ofte)